test/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -g")
 
-add_executable(
+#[[add_executable(
```

Comment:
Can we uncomment these tests?

Reply:
I have changed some stuff in data structure, so these test will not compile right. :) I think I'll fix them or remove them altogether.

Reply:
:+1: for fixing

I can help if need be. Also might make sense to split them into a simple Eigen, Blaze, and PETSC tests.